spec/features/unlinked_defendant_flow_spec.rb

```diff
@@ -254,7 +254,7 @@ def then_has_offence_details(table_number: 1)
       expect(page).to have_css('.govuk-table__header', text: 'Mode of trial')
 
       expect(page).to have_css('.govuk-table__cell', text: 'Not guilty on 12/04/2020')
-      expect(page).to have_css('.govuk-table__cell', text: /Indictable only:.*Court directs trial by jury/)
+      expect(page).to have_css('.govuk-table__cell', text: /Indictable only: Court directs trial by jury/)
```

Comment:
wondering if the designs want a line break between mode of trial and reasons. thats what the `.*` was here to capture originally

Reply:
👍 There was a line break between mode of trial and reasons - have reinstated the test for this.

Reply:
maybe we should make this explicit test for the `<br>` but, like the view spec below, i am not sure what "text" is being rendered does a `<br>` come out in text as a `\n` or does it come out as empty text - which is implied by this test passing